### PR TITLE
Add 'handle' support.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: src/resources/ble_esp32.cc [hunk 1]
```diff
@@ -1820,6 +1820,12 @@ PRIMITIVE(write_value) {
   return Smi::from(with_response ? 1 : 0);
 }
 
+PRIMITIVE(handle) {
+  ARGS(Resource, resource)
+  auto element = reinterpret_cast<BleReadWriteElement*>(resource);
```
Comment (floitsch) [inline src/resources/ble_esp32.cc:1825]: The reinterpret_cast is horrible, but that's the pattern that is used throughout this file.
Needs to get fixed separately.

File: src/resources/ble_darwin.mm [hunk 2]
```diff
@@ -1207,6 +1207,10 @@ _new BleRemoteDeviceResource(
   UNIMPLEMENTED();
 }
 
+PRIMITIVE(handle) {
+  UNIMPLEMENTED();
```
Comment (kasperl) [inline src/resources/ble_darwin.mm:1211]: Maybe these primitives should be using `FAIL(UNIMPLEMENTED)`, so we don't crash the VM for no good reason?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: src/resources/ble_darwin.mm (hunk 2 above)
Comment (floitsch) [reply src/resources/ble_darwin.mm:1211]: I will do this in a follow-up PR.

**Pull request merged**